components/src/dynamo/vllm/multimodal_utils/threaded_micro_batcher.py

```
@@ -0,0 +1,468 @@
+            if self._max_batch_cost is not None and c > self._max_batch_cost:
+                raise ValueError(
+                    f"item cost {c} exceeds max_batch_cost {self._max_batch_cost}; "
+                    "it has no batch it can fit"
+                )
+        request: _Request = _Request(
+            completion=concurrent.futures.Future(),
+            results=[None] * len(items),
+            remaining=len(items),
+        )
+        works = [
+            _Work(item, c, request, i) for i, (item, c) in enumerate(zip(items, costs))
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)